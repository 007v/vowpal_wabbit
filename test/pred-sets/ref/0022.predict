-0.388504
-0.388504
-0.388507
0.437578
-0.388505
-0.388505
-0.388507
0.437579
