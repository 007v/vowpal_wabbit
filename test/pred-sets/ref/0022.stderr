only testing
predictions = 0022.predict
Num weight bits = 16
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/3parity
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
0.373927 0.373927            1            1.0  -1.0000  -0.3885        4
0.373927 0.373927            2            2.0  -1.0000  -0.3885        4
1.185609 1.997291            4            4.0  -1.0000   0.4376        4
1.161071 1.136534            8            8.0   1.0000   0.4376        4

finished run
number of examples per pass = 8
passes used = 1
weighted example sum = 8.000000
weighted label sum = 0.000000
average loss = 1.161071
best constant = 0.000000
best constant's loss = 1.000000
total feature number = 32
